EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,153 @@
+
+## Specification
+
+The canonical genesis file MUST be a JSON object with the following top-level fields:
+
+### Top-Level Fields
+
+| Field           | Type              | Required | Description                                                     |
+|-----------------|-------------------|----------|-----------------------------------------------------------------|
+| `config`        | `object`          | Yes      | Chain configuration parameters.                                 |
+| `alloc`         | `object`          | No       | Map of addresses to pre-allocated balances and/or code/storage. |
+| `blobSchedule`    | `object`          | No       | EIP-4844 DAS configuration parameters.                          |
+| `nonce`         | `string`          | No       | Hex-encoded nonce (8 bytes).                                    |
+| `timestamp`     | `string`          | No       | Hex-encoded UNIX timestamp.                                     |
+| `extraData`     | `string`          | No       | Arbitrary extra data (max 32 bytes for Ethereum Mainnet).       |
+| `gasLimit`      | `string`          | Yes      | Hex-encoded block gas limit.                                    |
+| `difficulty`    | `string`          | Yes      | Hex-encoded block difficulty.                                   |
+| `mixhash`       | `string`          | No       | Hex-encoded mix hash.                                           |
+| `coinbase`      | `string`          | No       | Hex-encoded address.                                            |
+| `gasUsed`       | `string`          | No       | Initial gas used, defaults to `"0x0"`.                          |
+| `baseFeePerGas` | `string`          | No       | Initial base fee per gas.                                       |
+
+### `config` Object
+
+The `config` object contains hardfork activation block numbers and fork configurations. Known keys include:
+
+| Field     | Type     | Description                                      |
+|-----------|----------|--------------------------------------------------|
+| `chainId` | `number` | unique identifier for the blockchain.            |
+| `<hardforkName>`| `number` | block height or timestamp to activate the named hardfork.|
+|`terminalTotalDifficulty`| `number` | difficulty after which to switch from PoW to PoS.|
+|`terminalTotalDifficultyPassed`| `boolean` | PoS at genesis or not |
+|`depositContractAddress` | `string` | Ethereum address for the deposit contract |
+
+### `blobSchedule` Object
+
+| Field    | Type     | Description                                  |
+|----------|----------|----------------------------------------------|
+| `target` | `number` | desired number of blobs to include per block |
+| `max` | `number` | maximum number of blobs to include per block |
+| `updateFraction` | `number` | input to pricing formula per EIP-4844 |
```

Comment:
I checked but this is indeed the key Geth uses. It does not match EIP-7840 :thinking: